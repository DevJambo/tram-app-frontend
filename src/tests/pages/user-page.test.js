import React from "react";
import { render } from "@testing-library/react";
import UserPage from "../../components/pages/user-page";

describe("UserPage", () => {
  const { asFragment } = render(<UserPage />);

  it("renders correctly", () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
