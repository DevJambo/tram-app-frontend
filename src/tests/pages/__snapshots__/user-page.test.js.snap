// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserPage renders correctly 1`] = `
<DocumentFragment>
  <div>
    <h2
      class="user-header"
    >
      User Page
    </h2>
  </div>
</DocumentFragment>
`;
